### ISSUE TITLE
no problems reported when given an absolute path to a null module

### ISSUE THREAD

**Issue by bhcleek** [golangci/golangci-lint#592]
#### reproduction
Try to lint a directory with files that are missing godoc in a null module with golint

##### demonstration
Here's a script to demonstrate the problem:
```sh
#!/bin/sh

cd $(mktemp -d)

cat <<EOF > lint.go
package lint

import "fmt"

func MissingDoc() {
	fmt.Println("missing doc")
}
EOF

printf "linting a file\n" >&2
golangci-lint run -v --print-issued-lines=false --disable-all --exclude-use-default=false --enable=golint $PWD/lint.go

printf "linting a directory in a null module\n" >&2
golangci-lint run -v --print-issued-lines=false --disable-all --exclude-use-default=false --enable=golint $PWD

go mod init lint

printf "linting a directory in a module\n" >&2
golangci-lint run -v --print-issued-lines=false --disable-all --exclude-use-default=false --enable=golint $PWD
```

#### expected result
missing godoc is correctly identified

#### actual result
no errors are reported

##### environment
1. Version of golangci-lint: f8a5a8cb0191e291c1dd07adb9632f553b4d5a70
2. Config file: none
3. Go environment:
```
go version go1.12.4 darwin/amd64
GOARCH="amd64"
GOBIN="/Users/bhcleek/go/bin"
GOCACHE="/Users/bhcleek/Library/Caches/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOOS="darwin"
GOPATH="/Users/bhcleek/go"
GOPROXY=""
GORACE=""
GOROOT="/usr/local/go"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/darwin_amd64"
GCCGO="gccgo"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/f9/d6g779t53gq_20w0zm2f_1zm0000gn/T/go-build592573986=/tmp/go-build -gno-record-gcc-switches -fno-common"
```

**Comment by bhcleek** [comment]
more activity. This needs to be fixed.
